packages/nimble-components/src/anchor/template.ts

```diff
@@ -43,19 +41,26 @@ AnchorOptions
         aria-relevant="${x => x.ariaRelevant}"
         aria-roledescription="${x => x.ariaRoledescription}"
         ${ref('control')}
+    >${
+    /* Start and End slot templates inlined to avoid extra whitespace.
+        See https://github.com/microsoft/fast/issues/6557 */ ''
+}${
+    /* Whitespace intentionally avoided between tags for inline styles */ ''
+}<span
```

Comment:
All the mixing of styles of no space between tags and space between tags makes this challenging to read. Can we use a consistent format of removing all the space between tags and an attribute per line

Reply:
I tried to format it more consistently. Hopefully it is easier to read now.